src/lib/resource/sections/applicant-do-you-have-disabling-mental-injury.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
                             description:
                                 "This means it's much harder than usual to do things you would normally do, like going to work, seeing friends, or having a relationship.",
```

Comment:
Can you change 'myself' hint text to: 
This means it’s much harder than usual to do things you would normally do, like going to school or university, seeing friends, working or having a relationship.